### PR TITLE
Feat: Meta Detection

### LINKED ISSUE #268: Disable Bounties for "Parent Issues"
### Context

It's fast and easy to write todo lists in the original comment of a GitHub issue, and then immediately convert them to "child issues" directly from the GitHub user interface.

The problem I am seeing is that the bounty system breaks down where contributors can "double dip" on rewards easily by claiming bounties for the actual bounties (child issues) and then the "meta" or "parent" issue.

The argument possibly could be made for project management. However in the future all of the **issue creators**, and **commenters** (both are how the project manager would "manage") are to receive compensation for their contributions upon the completion of the bounty.

### Proposal

- When changing a label, the bot should check if there are linked issues in the original comment and disable setting a price (and most importantly, as a consequence, payment permit generation.)

- The bot should comment explaining why it refuses to set a price.

---

              Parent issues should not have bounties. Instead, the child issues should have bounties.

_Originally posted by @pavlovcik in https://github.com/ubiquity/grants/issues/5#issuecomment-1519067539_

### PR DESCRIPTION
**Pull request by Steveantor**
resolves #268

### CODE REVIEW

**Review by pavlovcik — APPROVED** [review]
Reviewed from my phone quickly seems fine.

**Comment by Steveantor** [comment]
> Reviewed from my phone quickly seems fine. It just disables setting a price right? We should be able to set priority and time ideally but not 100% necessary if too complicated to implement for some reason. This just needs to avoid calculating a price label for the bounty.

Yes

File: src/handlers/pricing/action.ts [hunk 1]
```diff
@@ -19,6 +22,11 @@ export const pricingLabelLogic = async (): Promise<void> => {
 
   const targetPriceLabel = getTargetPriceLabel(minTimeLabel, minPriorityLabel);
   if (targetPriceLabel) {
+    const iM = await isMeta(payload.issue.title, payload.issue.body);
+    if (iM) {
+      logger.info(`Skipping labels since it's a meta issue`);
+      return;
+    }
```
Comment (pavlovcik) [inline src/handlers/pricing/action.ts:29]: Looks good to me.

**Review by pavlovcik — APPROVED** [review]

**Comment by pavlovcik** [comment]
@0xcodercrane will merge

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**